packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -1,21 +1,46 @@
 
-You can easily import any public repositories by pasting its GitHub URL into the “GitHub repository URL” field and hitting the “enter” key or the “Import button”. This action will automatically fork the repository and spin a development environment for you to start coding straight away.
+Upon logging in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar. On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button located in the top left corner of the screen.
 
-In addition, you can create a project from any repository (public or private if you are a Pro user) of any GitHub Organization you belong to or create a new project using one of the templates available.
+![Screenshot of the iOS Dashboard](/ios-dashboard.png)
 
-![IMG_01E76C8C3559-1.jpg](../images/IMG_01E76C8C3559-1.jpg)
\ No newline at end of file
+### Accessing Sandboxes and Repositories
+
+If you are an existing CodeSandbox user and after selecting a workspace, you will be presented with a list of sandboxes and repositories that you have recently interacted with. To view all available sandboxes and repositories, simply navigate through the different categories in the sidebar or use the **Search** functionality available at the top of the screen.
+
+### Creating a New Sandbox or Repository
+
+You also have the option of starting from scratch by creating a new sandbox or importing a repository from GitHub.
+
+#### Importing a Repository from GitHub
+
+To import a repository from GitHub, simply tap on the **New Repository** button to start the import process. If you haven't already, you'll be prompted to link your CodeSandbox account with a GitHub account.
+
+In the import screen, you can either enter the link to the GitHub repository you wish to import or select a repository from one of the GitHub organizations you belong to, including your personal GitHub organization. CodeSandbox will automatically create a fork in GitHub for repositories you do not have write permissions for.
+
+![Screenshot of the Import from GitHub screen](/ios-gh-import.png)
+
+#### Creating a Sandbox from a Template
+
+Creating a new sandbox using one of the official templates is another option. It's a quick and easy way to get started with any technology without any setup required.
+
+![Screenshot of the New Sandbox screen](/ios-new-sandbox.png)
+
+### Working with Local Sandboxes
+
+With CodeSandbox for iOS, you can also create small Node.js prototypes locally. This is a useful feature when you're developing on your iOS device and don't have internet access. However, the functionality of these local sandboxes is limited due to restrictions imposed by the iOS operating system on applications available from the App Store.
+
+For instance, local sandboxes are restricted to Node.js version 12, and any framework that relies on JIT compilation or WebAssembly won't work.
```

Comment:
We can add Cloud sandbox plug here. What do you think?
```suggestion
For instance, local sandboxes are restricted to Node.js version 12, and any framework that relies on JIT compilation or WebAssembly won't work.

In this case, we suggest you use CodeSandbox Cloud sandboxes on your computer if possible. 
```

Reply:
Good shout! although I won't mention your computer, I don't want the user to leave the iOS app 😛